Public/Src/Engine/Processes/SandboxedProcessPipExecutor.cs

```diff
@@ -2779,6 +2780,44 @@ private bool GetSpecialCaseRulesForSpecialTools(AbsolutePath processPath, Absolu
             return false;
         }
 
+        private Dictionary<AbsolutePath, bool> m_isDirSymlinkCache = new Dictionary<AbsolutePath, bool>();

```

Comment:
> private Dictionary<AbsolutePath, bool> m_isDirSymlinkCache = new Dictionary<AbsolutePath, bool>(); [](start = 7, length = 99)

Is it worth pooling this instance?

Reply:
not sure, what do you think?

Reply:
I don't have a good sense of how much churn this is going to get... or how likely is to store stuff on this structure for arbitrary builds.
Feel free to ignore this comment if you don't have a strong feeling about it.